src/store/modules/wallet.js

```diff
@@ -157,14 +157,19 @@ const getters = {
     state.events[0]?.timestamp &&
     state.eventLastSeenTs < new Date(state.events[0]?.timestamp).getTime(),
   getNotificationCount: (state, getters) => {
-    if (!state.eventLastSeenTs || !getters.getEvents || !getters.loginAddress) {
+    const { getEvents } = getters;
+    if (!state.eventLastSeenTs || !getEvents || !getters.loginAddress) {
       return 0;
     }
-    return getters.getEvents.filter(
-      e =>
+    return getEvents.reduce((count, e) => {
+      if (
         state.eventLastSeenTs < new Date(e.timestamp).getTime() &&
-        (e.eventType === 'nft_sale' || e.eventType === 'receive_nft')
-    ).length;
+        ['nft_sale', 'receive_nft', 'mint_nft'].includes(e.eventType)
```

Comment:
Aren't we using new_class event though

Reply:
have formatted [eventType](https://github.com/likecoin/liker-land/blob/develop/src/store/modules/wallet.js#L180-L181)

Reply:
Maybe the event type should be declared as constant for reuse